README.md

```diff
@@ -1,6 +1,7 @@
 # Agentbaker
 
-[![Coverage Status](https://coveralls.io/repos/github/Azure/AgentBaker/badge.svg?branch=mikolaj/add-coveralls)](https://coveralls.io/github/Azure/AgentBaker?branch=mikolaj/add-coveralls)
+[![Coverage Status](https://coveralls.io/repos/github/Azure/AgentBaker/badge.svg?branch=master/add-coveralls)](https://coveralls.io/github/Azure/AgentBaker?branch=master/add-coveralls)

```

Comment:
wait, shouldn't it be branch="master", not "master/add-coveralls"?

Reply:
Yeah good point, coveralls website doesn't have any runs on master yet I think, thats why it suggested adding badge from my branch.

Reply:
that's fine, the badge should be empty initially, but Cameron is right the branch name is still wrong I missed that

Reply:
Correctly points to master now